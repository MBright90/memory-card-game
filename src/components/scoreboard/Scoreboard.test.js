import Scoreboard from '@components/scoreboard'
import { render, screen } from '@testing-library/react'

test('renders current score scoreboard', () => {
  render(<Scoreboard />)
  const scoreParagraphElement = screen.getByText(/score: ([0-9]|1[012])/i)
  expect(scoreParagraphElement).toBeInTheDocument()
})

test('renders best score scoreboard', () => {
  render(<Scoreboard />)
  const bestParagraphElement = screen.getByText(/best: ([0-9]|1[012])/i)
  expect(bestParagraphElement).toBeInTheDocument()
})
