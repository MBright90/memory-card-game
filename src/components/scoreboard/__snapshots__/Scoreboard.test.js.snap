// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scoreboard renders to match snapshot 1`] = `
<div>
  <div
    class="scoreboard"
    role="presentation"
  >
    <p
      style="color: rgb(255, 255, 255);"
    >
      Score: 
      3
    </p>
    <p
      style="color: rgb(255, 255, 255);"
    >
      Best: 
      5
    </p>
  </div>
</div>
`;
